src/Uno.UI.Runtime.Skia.Gtk/Extensions/Storage/Pickers/FileOpenPickerExtension.cs

```diff
@@ -77,6 +77,7 @@ private async Task<StorageFile[]> OpenPickerAsync(bool multiple)
 				}
 			}
 
+			dialog.Dispose();
```

Comment:
Good find! It would make sense to do both in a try/finally (in case there's an exception in between. Do you know if the fix works as well when only dispose is invoked ? If so, maybe doing this will be easier:

```csharp
using FileChooserDialog dialog = new FileChooserDialog(
```

The same question goes for the two other modifications.

Reply:
Very nice. Thanks!